Block/AccountBlockService.php

```diff
@@ -25,7 +25,7 @@
  *
  * @author Thomas Rabaix <thomas.rabaix@sonata-project.org>
  */
-class AccountBlockService extends BaseBlockService
+class AccountBlockService extends AbstractBlockService
```

Comment:
I'm not very familiar with the block bundle, what would make a block embeddable? The `getBlockMetadata` method ?